rend3-routine/shaders/src/opaque.wgsl

```diff
@@ -435,10 +437,7 @@ fn get_pixel_data(material: Material, vs_out: VertexOutput) -> PixelData {
 }
 {{/if}}
 
-fn surface_shading(light: DirectionalLight, pixel: PixelData, view_pos: vec3<f32>, occlusion: f32) -> vec3<f32> {
-    let view_mat3 = mat3x3<f32>(uniforms.view[0].xyz, uniforms.view[1].xyz, uniforms.view[2].xyz);
-    let l = normalize(view_mat3 * -light.direction);
-
+fn surface_shading(l: vec3<f32>, intensity: vec3<f32>, pixel: PixelData, view_pos: vec3<f32>, occlusion: f32) -> vec3<f32> {
```

Comment:
```suggestion
fn surface_shading(light_dir: vec3<f32>, intensity: vec3<f32>, pixel: PixelData, view_pos: vec3<f32>, occlusion: f32) -> vec3<f32> {
```

I know the old code used `l`, but now that the outside world needs to think about it, we should give it a descriptive name.